pkg/apis/core/v1alpha1/zz_generated.kubebuilder.go

```diff
@@ -303,7 +303,6 @@ var (
 			Scope: "Namespaced",
 			Validation: &v1beta1.CustomResourceValidation{
 				OpenAPIV3Schema: &v1beta1.JSONSchemaProps{
-					Type: "object",
```

Comment:
I think this change will not needed if the PR https://github.com/kubernetes-sigs/federation-v2/pull/250 got merged before this PR.

Reply:
Agree. if it lands before. i will remove it.